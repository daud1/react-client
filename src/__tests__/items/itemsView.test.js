import React from 'react';
import { shallow, render, mount } from 'enzyme';
import ItemsView from '../../components/items/itemsView.component';
import shallowToJson from 'enzyme-to-json';

const match = {
    params: {
        list_id: 1
    }
}
describe ('ItemView Component', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<ItemsView match={match} />);
        expect(wrapper.exists(<div></div>)).toBe(true)
    });

    it('matches snapshot', () => {
        const wrapper = shallow(<ItemsView match={match} />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
})