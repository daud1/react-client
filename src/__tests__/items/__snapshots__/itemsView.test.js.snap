// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemView Component matches snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-default"
  >
    <div
      class="container-fluid"
    >
      <div
        class="navbar-header"
      >
        <Link
          className="navbar-brand"
          replace={false}
          to="/lists"
        >
          Shopping Lists
        </Link>
      </div>
      <div
        class="collapse navbar-collapse"
        id="bs-example-navbar-collapse-1"
      >
        <ul
          class="nav navbar-nav"
        >
          <Search
            callback={[Function]}
            listId={1}
          />
        </ul>
        <ul
          class="nav navbar-nav navbar-right"
        >
          <br />
          <withRouter(LogOut) />
        </ul>
      </div>
    </div>
  </nav>
  <CreateItem
    callback={[Function]}
    list_id={1}
  />
  <Lister
    callback={[Function]}
  />
  <nav>
    <ul
      className="pagination"
    />
  </nav>
</div>
`;
